packages/kos-sdk/src/chains/tron/requests.rs

```diff
@@ -2,6 +2,15 @@ use crate::utils;
 use kos_types::error::Error;
 use serde_json::json;
 
+use serde::Serialize;
+
+#[derive(Serialize)]
+pub struct CreateTRC20TransferOptions {
```

Comment:
struct name should notba an action, in this case use only `TransferOptions`, it also should be compatible with other chains, please check options used in other chains